src/docs/Components/Buttons/accessibility.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Accessibility'
+order: 4
+---
+
+## Accessibility and semantic
+
+<HintItem>
+  Alway wrap your label into a <code>span.mc-button__label</code> tag
+</HintItem>
+
+<HintItem>
+  Use <code>aria-label="button purpose"</code>{' '}
+  <b>using only an icon without text</b>.
+</HintItem>
+
+```html
+<a
+  href="#"
+  class="mc-button mc-button--icon-only mc-button--icon-medium-m"
+  aria-label="my button purpose"
+>
+  <!-- Your icon html tag -->
+</a>
+```
```

Comment:
Please could you update this code, by the following:

```html
<button
  type="button"
  class="mc-button mc-button--icon-only mc-button--icon-medium-m"
  aria-label="my button purpose"
>
  <!-- Your icon html tag -->
</button>
```

Reply:
2a5dc61